ocamldebug.el

```diff
@@ -473,12 +476,18 @@ the ocamldebug commands `cd DIR' and `directory'."
     (setq default-directory (file-name-directory path))
     (message "Current directory is %s" default-directory)
     (setq ocamldebug-command-name
-	  (read-from-minibuffer "OCaml debugguer to run: "
-				ocamldebug-command-name))
-    (make-comint (concat "ocamldebug-" file)
-		 (substitute-in-file-name ocamldebug-command-name)
-		 nil
-		 "-emacs" "-cd" default-directory path)
+          (read-from-minibuffer "OCaml debugger to run: "
+                                ocamldebug-command-name))
+    (setq ocamldebug-debugee-args
+          (read-from-minibuffer "Arguments to the debuggee: "
```

Comment:
Can't these be read on the same line as `ocamldebug-command-name`?  One does not need two prompts, do we.  (I would prefer a single prompt just as `C-c C-c` does for make.)